packages/shared/src/components/search/SearchBar.tsx

```diff
@@ -19,7 +20,7 @@ export function SearchBar({
   isLoading,
   ...props
 }: SearchBarProps): ReactElement {
-  const suggestionsProps = useSearchSuggestions();
+  const suggestionsProps = useSearchSuggestions({ origin: Origin.HomePage });
```

Comment:
Search bar can show up on the search page as well, isn't it?

Reply:
Ah good spot, yes it can!

Reply:
yes it does, however this distinction was not made in the taxonomy sheet so I ignored it -- do you want to keep track of this? -- IMO not sure how valuable is the information of knowing a suggestion showed up on the home page or search page